assets/models/system/databricks-dolly-v2-12b/spec.yaml

```diff
@@ -16,5 +16,10 @@ tags:
   - Standard_ND40rs_v2
   - Standard_ND96asr_v4
   license: mit
+  model_specific_defaults:
```

Comment:
Are these settings needed for deployment?

Reply:
Yes Sasi

Reply:
@ShipraJain01 Are the "model_specific_defaults" settings used for evaluation?